Jellyfin.Api/Controllers/ImageController.cs

```diff
@@ -84,6 +84,7 @@ public class ImageController : BaseJellyfinApiController
         /// <returns>A <see cref="NoContentResult"/>.</returns>
         [HttpPost("Users/{userId}/Images/{imageType}")]
         [HttpPost("Users/{userId}/Images/{imageType}/{index?}", Name = "PostUserImage_2")]
+        [Authorize(Policy = Policies.DefaultAuthorization)]
```

Comment:
`DeleteUserImage` is missing as well. Can you re-check the entire controller for auth discrepancies?

Reply:
this was the only one missing thankfully